example/issue_comment.go

```diff
@@ -17,13 +17,29 @@ package main
 import (
 	"context"
 	"encoding/json"
+	"encoding/xml"
 	"fmt"
+	"regexp"
 	"strings"
+	"time"
 
 	"github.com/google/go-github/v56/github"
 	"github.com/palantir/go-githubapp/githubapp"
 	"github.com/pkg/errors"
+	"github.com/redhat-appstudio/qe-tools/pkg/prow"
 	"github.com/rs/zerolog"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	reporters "github.com/onsi/ginkgo/v2/reporters"
+)
+
+const (
+	targetAuthor  = "dheerajodha"
```

Comment:
I guess this will change after we move the repo to redhat-appstudio org

Reply:
Yes, planning to do this as soon I get the confirmation that the rest of the code is ready to get merged.